14_functions_lab/05.py

```diff
@@ -1,12 +1,19 @@
-"""
-Write a function called "groupby" that takes a list
-and a function and returns a dictionary
-keyd by the return value of the function on the list items
-
-For example:
-    groupby(lambda s: s[0], ['foo', 'fi', 'hello', 'hi'])
-    returns: { 'f': ['foo','fi'], 'h': ['hello', 'hi'] }
-"""
-
-def groupby(f, words):
-    pass
+#!/usr/bin/python
+
+"""this list is just here for the test"""
+
+stateList = ['Arizona', 'Cleveland', 'Alabama', 'Chicago', 'Los Angeles', 'Louisiana']
+
```

Comment:
Very well. Also try to read about defaultdict which already does the if functionality you used:
https://docs.python.org/2/library/collections.html#collections.defaultdict